Doppler.Sap/Models/SapSaleOrderInvoiceResponse.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.Sap.Models
+{
+    public class SapSaleOrderInvoiceResponse
+    {
+        public string CardCode { get; set; }
+        public int DocEntry { get; set; }
+        public DateTime DocDate { get; set; }
```

Comment:
This property is not used, can you remove it?

Reply:
Now is used in the mapper.